workflow/rules/commons/02_pyutils.smk

```diff
@@ -14,6 +15,17 @@ def logout(msg):
     return
 
 
+def get_username():
+    user = os.getlogin()
```

Comment:
I just learned about a preferred library function in Python3 for this purpose - can you please update this function using that library:
`getpass.getuser()`